api-public/src/main/kotlin/gropius/graphql/GropiusGraphQLContextFactory.kt

```diff
@@ -39,4 +56,22 @@ class GropiusGraphQLContextFactory(
         return super.generateContextMap(request) + additionalContextEntries
     }
 
+    /**
+     * Verifies a jwt and returns the [GropiusUser] defined by the id in subject
+     *
+     * @param token the jwt, possibly starting with `"Bearer "`
+     * @return the user defined by the id in subject
+     * @throws IllegalStateException if the user does not exist or if the jwt is invalid
+     */
+    private suspend fun verifyToken(token: String): GropiusUser {
+        val tokenWithoutBearer = token.replace("Bearer ", "")
```

Comment:
Must be case insensitive matching for the auth scheme, see https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Authorization#directives

Reply:
Maybe as idea in future for easier integation with a frontend, we could think about supporting tokens in cookie header.

Reply:
69d97778ef86e49b34a43e423b0f290a463c5073